packages/manager/src/features/CloudPulse/Alerts/AlertsListing/AlertTableRow.tsx

```diff
@@ -0,0 +1,46 @@
+import { Typography } from '@linode/ui';
+import * as React from 'react';
+
+import { DateTimeDisplay } from 'src/components/DateTimeDisplay';
+import { TableCell } from 'src/components/TableCell';
+import { TableRow } from 'src/components/TableRow';
+import { capitalize } from 'src/utilities/capitalize';
+
+import { alertSeverityOptions } from '../constants';
+import { AlertActionMenu } from './AlertActionMenu';
+
+import type { Alert } from '@linode/api-v4';
+
+interface Props {
+  alert: Alert;
+}
+
+export const AlertTableRow = React.memo((props: Props) => {
+  const { alert } = props;
+  const { created_by, id, label, service_type, status, updated } = alert;
+  const alertSeverity = alertSeverityOptions.find(
+    (option) => option.value === alert.severity
+  );
```

Comment:
React memo can be removed from the whole component and can be added for alertSeverity